src/prefect/utilities/filesystem.py

```diff
@@ -9,6 +9,7 @@
 
 import fsspec
 import pathspec
+import prefect
```

Comment:
I don't know if this is going to cause a circular import problem? 🤔 

That might have been why we were using `Parent(__file__).parent.parent` previously